app/Utils/feverUtil.php

```diff
@@ -1,69 +1,62 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
This will just make the application crash in production if any type is not strictly as declared, which is most likely a bad idea at the moment.
Let's come back to that when we have a higher score on PHPStan https://github.com/FreshRSS/FreshRSS/issues/4112

Reply:
https://dev.to/rocksheep/the-way-stricttypes-works-in-php-eb7
In this class, there is no problem